draft-ietf-quic-transport.md

```diff
@@ -2860,12 +2860,13 @@ valid frames? -->
 ### Sending ACK Frames
 
 An endpoint MUST NOT send more than one packet containing only an ACK frame per
-received packet that contains frames other than ACK and PADDING frames.
-An endpoint MUST NOT send a packet containing only an ACK frame in response
-to a packet containing only ACK or PADDING frames, even if there are packet
-gaps which precede the received packet. This prevents an indefinite feedback
-loop of ACKs. The endpoint MUST however acknowledge packets containing only
-ACK or PADDING frames when sending ACK frames in response to other packets.
+received ACK-eliciting packet(one containing frames other than ACK and/or
+PADDING).  An endpoint MUST NOT send a packet containing only an ACK frame in
```

Comment:
s/MUST NOT send a packet containing only an ACK frame/MUST NOT send a non-ack-eliciting packet/.

Reply:
yeah I thought of that one - but didn't say anything because you are discussing send ACK's. But to correct yes, it needs to include PAD.

Reply:
I'm not sure that change is correct.  ACK only packets don't count towards congestion control, so they're somewhat special.  I'd be curious to have @janaiyengar opinion on this.